libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -76,11 +88,6 @@ export class ExecuteBridgeRequest {
       );
     }
 
-    const secretKey = await this.getDecryptedSecretKey.execute(
-      GetDecryptedSecretKeyCommand.create({
-        environmentId: command.environmentId,
-      }),
-    );
```

Comment:
Refactoring many of these to discrete methods to make the `execute` method easier to read.